pom.xml

```diff
@@ -10,6 +10,8 @@
   <version>1.0.0-ALPHA-10-SNAPSHOT</version>
   <properties>
     <stargate.version>${project.parent.version}</stargate.version>
+    <!-- 17-May-2023, tatu: [json-api#172] Need Jackson 2.15.x -->
+    <jackson.version>2.15.2</jackson.version>
```

Comment:
Quarkus 3.1.x will likely upgrade, but until we use that, override since we need this version for length limit validation.

Reply:
Just checked with 3.1.0, it's still on `2.15.0`..